compiler/testData/codegen/box/regressions/kt18779.kt

```diff
@@ -1,5 +1,3 @@
-// IGNORE_BACKEND_FIR: JVM_IR
-
```

Comment:
@mglukhikh Just in case, improving type info around `try` expression can unmute a bb test.